example/sampleuser/user.go

```diff
@@ -90,6 +90,10 @@ func (u *SampleUser) GetChannelStats(channelId string) error {
 	return nil
 }
 
+func (u *SampleUser) GetUsersStatusesByIds(userIds []string) ([]*model.Status, error) {
```

Comment:
We are missing this change and we are good. If you run `make test` you should see an error caused by this I believe.

Reply:
Sorry haste makes waster :\